packages/framework/fluid-framework/api-report/fluid-framework.alpha.api.md

```diff
@@ -793,6 +821,12 @@ export class SchemaFactory<out TScope extends string | undefined = string | unde
     readonly string: TreeNodeSchema<"com.fluidframework.leaf.string", NodeKind.Leaf, string, string>;
 }
 
+// @alpha
+export interface SchemaValidationFunction<Schema extends TSchema> {
+    // (undocumented)
```

Comment:
Docs? :)

Reply:
It has a `@returns` doc comment. I guess that doesn't work well with our tooling so I'll make it a "Returns" instead.

Reply:
Done.